src/main.rs

```diff
@@ -93,3 +101,64 @@ fn main() {
         thread::sleep(time::Duration::from_millis(SLEEP_TIME_MS));
     }
 }
+
+fn change_gain(increase: bool) {
+    // Retrieve current gain
+    let current_gain = get_gain();
+    let mut new_gain = current_gain;
+    if increase {
+        new_gain += GAIN_OFFSET;
+    } else {
+        new_gain -= GAIN_OFFSET;
+    }
+    // Clamp to make sure the gain is in the correct range
+    let clamped_gain = clamp(new_gain, MINIDSP_GAIN_MIN, MINIDSP_GAIN_MAX);
```

Comment:
We call `camp(...)` twice, every time using `MINIDSP_GAIN_MIN, MINIDSP_GAIN_MAX`. I think that should be removed and simply used directly.